client/question/components/LearnerViewDirective.js

```diff
@@ -25,79 +25,92 @@ tie.directive('learnerView', [function() {
-                  <div class="tie-next-curtain-container" ng-if="nextButtonIsShown">
+                  <div class="tie-next-curtain-container"
+                      ng-if="nextButtonIsShown">
                     <div class="tie-next-curtain"></div>
                     <div class="tie-arrow-highlighter"></div>
                     <div ng-click="showNextTask()" class="tie-next-arrow">
                       <span class="tie-next-button-text">Next</span>
                     </div>
                   </div>
-                  <div ng-if="codeEditorIsShown" class="tie-codemirror-container">
+                  <div ng-if="codeEditorIsShown"
+                      class="tie-codemirror-container">
                     <ui-codemirror ui-codemirror-opts="codeMirrorOptions"
-                                   ng-model="editorContents.code"
-                                   ng-change="onCodeChange()"
-                                   class="protractor-test-code-input">
+                        ng-model="editorContents.code"
+                        ng-change="onCodeChange()"
+                        class="protractor-test-code-input">
                     </ui-codemirror>
                   </div>
                 </div>
-                <select ng-if="SERVER_URL" class="tie-select-menu" name="lang-select-menu">
+                <select ng-if="SERVER_URL" class="tie-select-menu"
+                    name="lang-select-menu">
                   <option value="Python" selected>Python</option>
                 </select>
                 <a ng-if="!SERVER_URL" class="tie-primer-link tie-python-primer" target="_blank" ng-href="{{getPythonPrimerUrl()}}">New to python?</a>
-                <button class="tie-code-reset tie-button protractor-test-reset-code-btn" name="code-reset"
-                    ng-click="resetCode()">
+                <button class="tie-code-reset tie-button protractor-test-reset-code-btn" name="code-reset" ng-click="resetCode()">
                   Reset Code
                 </button>
-                <div class="tie-code-auto-save" ng-show="autosaveTextIsDisplayed">
+                <div class="tie-code-auto-save"
+                    ng-show="autosaveTextIsDisplayed">
                   Saving code...
                 </div>
                 <button class="tie-run-button tie-button tie-button-green"
                     ng-class="{'active': !nextButtonIsShown}"
                     ng-click="submitCode(editorContents.code)"
                     ng-disabled="nextButtonIsShown">
-                  I think I'm done
+                  I think I&#39m done
                 </button>
                 <button class="tie-run-button tie-button tie-button-blue protractor-test-run-code-btn"
                     ng-class="{'active': !nextButtonIsShown}"
                     ng-click="submitCode(editorContents.code)"
                     ng-disabled="nextButtonIsShown">
-                  I need help
+                  Check my code
```

Comment:
"I need help" and "Check my code" have different meanings to me, and the latter feels more like "I think I'm done" IMO. Just a data point.

Reply:
This is intentional. Currently, there is functionally no difference between the two buttons. Having greater contrast in the button labels makes this more obvious in a way that feels like a feature is missing or broken. While "I think I'm done" implies a greater level of confidence in having a complete answer than "Check my code", they are close enough so that the functional equivalence is less jarring. Otherwise, I suggest we make "I need help" functionally distinct.

Reply:
OK, let's go with "Check my code" for now -- I think the rationale makes sense, and thanks for bringing it up!

I think the main thing I'd want to check for during user studies is whether users are still reticent to click the "Check my code" button because they haven't finished solving the problem, which was what the "I need help" button was meant for in the first place. If we see this behaviour then perhaps we can try switching and see what happens.